packages/commonwealth/server/migrations/20230421205344-add_thread_link.js

```diff
@@ -2,10 +2,68 @@
 
 module.exports = {
   up: async (queryInterface, Sequelize) => {
-    return queryInterface.addColumn('Threads', 'links', {
+    await queryInterface.addColumn('Threads', 'links', {
```

Comment:
Migration should be transactionalized. Note that this makes the loop here tricky, as we will need to lock the entire table for the duration of the migration. I wonder if there's a cleaner way we can make these updates...